Backend/nodeJS/src/modules/controllers/task-controller.js

```diff
@@ -0,0 +1,54 @@
+const shop = require('/home/user/Documents/Work/module3/Shopping list/nodeJS/src/db/models/task/Index');
```

Comment:
when exporting a model, use upper case for the variable name, for example `Shop`